packages/mobile/src/screens/app-screen/AppTabScreen.tsx

```diff
@@ -111,11 +88,22 @@ export const AppTabScreen = ({ baseScreen, Stack }: AppTabScreenProps) => {
     <Stack.Navigator
       screenOptions={screenOptions}
       screenListeners={{
-        state: e => {
-          // @ts-ignore: this is cool, but doesn't exist according to types
+        state: (
+          e: EventArg<
```

Comment:
if you dont mind putting this type somewhere, would make it a lil easier to read <3

Reply:
ugh ty for typing it